src/interfaces/IOperationData.ts

```diff
@@ -13,5 +14,9 @@ export interface IOperationData {
       sender_public_key: string;
     };
     signature: string;
+    content_creator_pub_key: string;
+    content_creator_address: string;
+    id: string; //
```

Comment:
either fill the comment or remove `//`

```suggestion
    id: string;
```

Reply:
annotated